EUROTHRM/EUROTHRM-IOC-01App/Db/devEurothrm.db

```diff
@@ -270,46 +270,137 @@ record(ao, "$(P)RAMP_MAXOUT")
 
 #Fanout reads so that they can be interrupted
 
-record(fanout, "$(P)READ") {
-	field(SCAN, "5 second")
-	field(LNK1, "$(P)P PP")
-	field(LNK2, "$(P)I PP")
-	field(LNK3, "$(P)D PP")
-	field(LNK4, "$(P)OUTPUT PP")	
-	field(LNK5, "$(P)MAX_OUTPUT PP")
-	field(LNK6, "$(P)READ1 PP")
+record(seq, "$(P)READ") {
+
+	field(SCAN, "Passive")
+	field(DO1, "1")
+	field(DO2, "1")
+	field(DO3, "1")
+	field(DO4, "1")
+	field(DO5, "1")
+	field(DO6, "1")
+	field(DO7, "1")
+	field(DO8, "1")
+	field(DO9, "1")
+	field(DOA, "1")
+	
+	field(DLY1, "$(TREAD)")
+	field(DLY2, "$(TREAD)")
+	field(DLY3, "$(TREAD)")
+	field(DLY4, "$(TREAD)")
+	field(DLY5, "$(TREAD)")
+	field(DLY6, "$(TREAD)")
+	field(DLY7, "$(TREAD)")
+	field(DLY8, "$(TREAD)")
+	field(DLY9, "$(TREAD)")
+	field(DLYA, "$(TREAD)")
+	
+	field(LNK1, "$(P)READ_FAST.PROC")
+	field(LNK2, "$(P)READ_SLOW1.PROC")
+	field(LNK3, "$(P)READ_FAST.PROC")
+	field(LNK4, "$(P)READ_SLOW2.PROC")
+	field(LNK5, "$(P)READ_FAST.PROC")
+	field(LNK6, "$(P)READ_SLOW3.PROC")
+	field(LNK7, "$(P)READ_FAST.PROC")
+	field(LNK8, "$(P)READ_SLOW4.PROC")
+	field(LNK9, "$(P)READ_FAST.PROC")
+	field(LNKA, "$(P)READ_SLOW5.PROC")
+	
 	field(SELM, "All")
 }
 
-record(fanout, "$(P)READ1") {
+record(seq, "$(P)READ_RESTART"){
+	field(SCAN, "1 second")
+	field(LNK1, "$(P)READ.PROC")
+}
+
+record(seq, "$(P)READ_FAST") {
 	field(SCAN, "Passive")
-	field(LNK1, "$(P)AUTOTUNE PP")
-	field(LNK2, "$(P)SENSOR PP")
-	field(LNK3, "$(P)RBV PP")
-	field(LNK4, "$(P)SP:RBV PP")
-	field(LNK5, "$(P)HILIM PP")
-	field(LNK6, "$(P)LOWLIM PP")	
+	field(DO1, "1")
+	field(DO2, "1")
+	field(DLY1, "$(TDLY)")
+	field(DLY2, "$(TDLY)")
 	field(SELM, "All")
+	field(LNK1, "$(P)RBV.PROC")
+	field(LNK2, "$(P)OUTPUT.PROC")
+}
+
+record(seq, "$(P)READ_SLOW1") {
+	field(SCAN, "Passive")
+	field(DO1, "1")
+	field(DO2, "1")	
+	field(DLY1, "$(TDLY)")
+	field(DLY2, "$(TDLY)")
+	field(SELM, "All")
+	field(LNK1, "$(P)TEMP:HIGHLIMIT.PROC")	
+	field(LNK2, "$(P)TEMP:LOWLIMIT.PROC")		
+}
+
+record(seq, "$(P)READ_SLOW2") {
+	field(SCAN, "Passive")
+	field(DO1, "1")
+	field(DO2, "1")
+	field(DLY1, "$(TDLY)")
+	field(DLY2, "$(TDLY)")
+	field(SELM, "All")	
+	field(LNK1, "$(P)TEMP:SP:RBV:CONV.PROC")	
+	field(LNK2, "$(P)SP:RBV.PROC")
```

Comment:
Don't do both of these instead forward link the TEMP:SP:RBV:CONV from the SP:RBV record so that it is processed once the change is in.